vulture_os/toolkit/api_parser/cisco_umbrella_managed_org/cisco_umbrella_managed_org.py

```diff
@@ -0,0 +1,269 @@
+        if self.cisco_umbrella_managed_org_dns:
+            log_types.append("dns")
+        if self.cisco_umbrella_managed_org_proxy:
+            log_types.append("proxy")
+        try:
+            customer_ids_available = self._get_customers_id()
+            logger.info(f"[{__parser__}]:execute: Available customers ids are {customer_ids_available}", extra={'frontend': str(self.frontend)})
+        except Exception as e:
+            logger.error(f"[{__parser__}]:execute: could not get the list of organization ids: {e}", extra={'frontend': str(self.frontend)})
+
+        if self.cisco_umbrella_managed_org_customers_id:
+            customer_ids = set(self.cisco_umbrella_managed_org_customers_id.split(",")).intersection(set(customer_ids_available))
+        else:
+            customer_ids = set(customer_ids_available)
+        customer_ids = list(map(str, customer_ids))
+
+        for customer_id in customer_ids:
+            if not self.cisco_umbrella_managed_org_customers_tokens.get(customer_id) or not self.cisco_umbrella_managed_org_customers_tokens[customer_id].get("expires_at") or not self.cisco_umbrella_managed_org_customers_tokens[customer_id].get("access_token") or (self.cisco_umbrella_managed_org_customers_tokens[customer_id]["expires_at"] - timedelta(seconds=10) > timezone.now()):
+                try:
+                    self._get_customer_token(customer_id)
+                except Exception as e:
+                    logger.error(f"[{__parser__}]:execute: {e}", extra={'frontend': str(self.frontend)})
+            access_token = self.cisco_umbrella_managed_org_customers_tokens[customer_id]["access_token"]
+            self._connect(access_token)
+            for log_type in log_types:
+                timestamp_field_name = f"app{customer_id}_{log_type}"
+                since = self.last_collected_timestamps.get(timestamp_field_name) or (timezone.now() - timedelta(days=1))
+                to = min(timezone.now(), since + timedelta(hours=24))
+                logger.info(f"[{__parser__}]:execute: getting {log_type} logs from {since} to {to}, for organization {customer_id}", extra={'frontend': str(self.frontend)})
+                try:
+                    logs = self.get_logs(customer_id, log_type, since, to)
+                except Exception as e:
+                    logger.error(f"[{__parser__}]:execute: {e}", extra={'frontend': str(self.frontend)})
+                index = 0
+                logs_count = self.LIMIT_MAX
+                while logs_count == self.LIMIT_MAX and index <= self.OFFSET_MAX:
+                    try:
+                        response = self.get_logs(customer_id, log_type, since, to)
+                    except Exception as e:
+                        logger.error(f"[{__parser__}]:execute: {e}", extra={'frontend': str(self.frontend)})
+                    # Downloading may take some while, so refresh token in Redis
+                    self.update_lock()
+                    logs = response['data']
```

Comment:
```suggestion
                        logs = self.get_logs(customer_id, log_type, since, to)
                    except Exception as e:
                        logger.error(f"[{__parser__}]:execute: {e}", extra={'frontend': str(self.frontend)})
                    # Downloading may take some while, so refresh token in Redis
                    self.update_lock()
```